src/Functions/FunctionsJSON.h

```diff
@@ -1614,7 +1614,7 @@ class JSONExtractRawImpl
         }
         if (element.isDouble())
         {
-            writeFloatText(element.getDouble(), buf);
+            writeFloatText(element.getDouble(), buf, true);
```

Comment:
This change is controversial, or we can make controll it by a setting as we pass settings to JSON formatter.
I will remove it after tests run.